MediaGallerySynchronizationApi/Model/GetContentHashInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MediaGallerySynchronizationApi\Model;
+
+/**
+ * Get hashed value of image content.
+ */
+interface GetContentHashInterface
+{
+    /**
+     * Get hashed value of image content.
+     *
+     * @param string $filepath
+     * @return string
+     */
+    public function execute(string $filepath): string;
```

Comment:
```suggestion
    public function execute(string $content): string;
```

It would be better to work with file content instead of the file (we wouldn't like to include file path/name in the hash) as it will not allow us to use this hash for duplicates detection

Reply:
If we're gonna go with hashing the file content, then we'll go with `sha1()` instead of `sha1_file()` since the latter uses file path as a parameter instead of the entire file content.